tests/unittests/server/utils/jsonpatch/test_jsonpatch_methods.py

```
@@ -251,6 +290,31 @@ def test_append_list_item(idx: Union[str, int], person: Person):
+        #####
+
+        assert doc.id == "new"
+        assert res is doc
+
+    @staticmethod
+    def test_setitem_keyerror_reraised_without_attribute():
+        """Tests that a ``KeyError`` from item access is re-raised (as
+        ``ValueError``) when there is no matching attribute to fall back to."""
+        doc = EmbeddedDoc()
+
+        with pytest.raises(ValueError):
```

Ruff (0.15.17)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)